docs/current_docs/configuration/engine.mdx

```diff
@@ -0,0 +1,394 @@
+---
+slug: /configuration/engine
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import { daggerVersion } from "../partials/version";
+
+# Engine
+
+Dagger is designed to be run out-of-the-box with sensible defaults.
+However, even with these defaults, it may sometimes be necessary to modify various
+settings.
+
+TODO: update version when base PR is merged
```

Comment:
@jedevc this looks ready to merge except for this TODO - do you want to remove it or is there something else required here?

Reply:
Yeah, we need to merge https://github.com/dagger/dagger/pull/8800 first.